amaru-node/src/main.rs

```diff
@@ -0,0 +1,37 @@
+use std::env::{set_var, var};
+use tracing::info;
+
+#[tokio::main]
+async fn main() {
+    init_logging();
+    info!("Hello, amaru-node!");
+}
+
+fn init_logging() {
+    match var("RUST_LOG") {
+        Ok(_) => {}
+        Err(_) => {
+            // set a default logging level of info if unset.
+            set_var("RUST_LOG", "info");
+        }
+    }
+
+    let tracing_filter = match var("RUST_LOG") {
+        Ok(level) => match level.to_lowercase().as_str() {
+            "error" => tracing::Level::ERROR,
+            "warn" => tracing::Level::WARN,
+            "info" => tracing::Level::INFO,
+            "debug" => tracing::Level::DEBUG,
+            "trace" => tracing::Level::TRACE,
+            _ => tracing::Level::INFO,
+        },
+        Err(_) => tracing::Level::INFO,
+    };
```

Comment:
we could probably use the following instead of manual mapping:

https://docs.rs/tracing-subscriber/latest/tracing_subscriber/filter/struct.EnvFilter.html#